src/overlays/actors/ovl_En_Syateki_Okuta/z_en_syateki_okuta.h

```diff
@@ -15,13 +47,13 @@ typedef struct EnSyatekiOkuta {
     /* 0x144 */ SkelAnime skelAnime;
     /* 0x188 */ EnSyatekiOkutaActionFunc actionFunc;
     /* 0x18C */ ColliderCylinder collider;
-    /* 0x1DC */ Vec3f unk_1D8;
+    /* 0x1DC */ Vec3f headScale;
     /* 0x1E4 */ Vec3s jointTable[OCTOROK_LIMB_MAX];
     /* 0x244 */ Vec3s morphTable[OCTOROK_LIMB_MAX];
-    /* 0x2A4 */ s16 unk_2A4;
-    /* 0x2A6 */ s16 unk_2A6;
+    /* 0x2A4 */ s16 deathTimer;
+    /* 0x2A6 */ s16 type;
     /* 0x2A8 */ UNK_TYPE1 unk_2A8[0x2];
-    /* 0x2AA */ s16 unk_2AA;
+    /* 0x2AA */ s16 circleOrCrossAlpha;
```

Comment:
I remember this got mentioned, and `hitmark` came up as a name to describe the circle/cross (would be nice to think of a name as well for the torch jumping minigame). I saw one opposition, but hitmark seems fine to me. Or maybe something like `minigameHitmark`?

Reply:
I didn't like it in this case because, when you hit them, an actual hitmark shows up too.
![GLideN64_ZELDA_MAJORA'S_MASK_009](https://github.com/zeldaret/mm/assets/12245827/9ebe1e89-d9d9-48c2-8d6f-6e74421b5f40)
I didn't want to cause too much confusion between the *actual* hitmark and the circle/cross that appears. That's why I went with the current name - it's as unambiguous as possible. I also think that, if the goal is synchronize this name with the jumping minigame one, then putting "hit" in the name is the opposite of what we want. In the jumping minigame, you don't really "hit" anything.

What do you think? I definitely think this name is a little unwieldy, but I tried to avoid using "hitmark" for a reason. Do you think I should go with "hitmark" anyways?

Reply:
Oh true hitmark doesn't really work for jumping minigame. A few other brainstorming ideas are `minigameMark` (just remove hit) or perhaps `scoreMark`, maybe mix the two `minigameScoreMark`

Reply:
Hmm, the thing with `scoreMark` is that only the circle actually corresponds to the score. When the circle appears, the score increases by one. When the cross appears, the timer goes down, and the score remains the same. I forgot what happens in the jumping minigame, but I think it's similar in the sense that the cross doesn't actually change the score.

For `minigameMark`, I think it's a little better, but it's also more ambiguous than what currently exists. `circleOrCrossAlpha` makes it pretty clear what it controls, but `minigameMarkAlpha` is a bit more vague. If you don't already know the specifics as to how the minigame works, it might be confusing what this actually changes.

Reply:
And just to add even more confusion to this, in MM3D, in North America and Europe, they changed the circle to a checkmark:
![image](https://github.com/zeldaret/mm/assets/12245827/cb65cd8d-a6e8-48ac-95ff-fec2e0a22c51)

So I think there *is* a better name for this...but I can't quite think of it. Because if this was ported to MM3D, we'd have to change the name of this to `circleCheckOrCrossAlpha`, which is a pretty crazy name.

I don't know if this helps, or if it adds even more confusion, but this is currently where I'm at. I think there's a name better than what I have, but I don't know what that name might be.

Reply:
What about `hitResultMark`? or just `hitResult`? `hitResultAlpha` as well

Reply:
That's true. I suppose just by nature of being different minigames, I'd be okay with `hitResult` here and `jumpResult` for the jumping game, but I can see why consistency would be nice. What about `eventResult` or just `resultMark`?

Reply:
I think `hitResult` is the best thing proposed so far. The thing with `eventResult` is that "event" in this codebase usually means something specific, and in this case it means something else, so I can see how it might cause confusion. For `resultMark`, it's a bit ambiguous. The result of what? It's the result of getting hit, hence why I think `hitResult` is the best thing so far.

However, I took a look at the jumping game stuff, because I was curious if *the developers* treated them as the same. And they totally did. The names are `jgame_maru_model` and `jgame_peke_model`, just like how the Octoroks have it. They even swapped the circle for a green checkmark in the US and EU releases of MM3D! 

![image](https://github.com/zeldaret/mm/assets/12245827/0746ac80-cdaa-4458-a858-523b9ac60474)

Now, we're not beholden to their conventions; we can do whatever we want. But at the same time, these are clearly the same thing, so it feels a bit odd to deviate. I'd be fine with either the current `circleOrCrossAlpha` or `hitResultAlpha` (and presumably `jumpResultAlpha` for the jumping game), but after all this discussion, I'm actually kind of leaning to the former, even if I wasn't that big of a fan to begin with. What do you think?

Reply:
Sure, let's go with that then.